api/sheets/get_name.md

```diff
@@ -6,40 +6,36 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the name of a node.
+# Get a node's name
 
-## `=DCGETNAME(dcids)`
+Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.
 
-Given a cell or a column range of cells with DCIDs, return the names of those dcids.
+## General information about this method
 
-**Arguments**
-*    `dcids` - DCIDs to get the names of
+**Signature**: `=DCGETNAME(dcids)`
```

Comment:
Should these be "Formula" instead for Sheets?

Reply:
Yep, that is a GREAT idea!